libsawtooth/src/client/rest.rs

```diff
@@ -47,11 +47,7 @@ impl SawtoothClient for RestApiSawtoothClient {
     fn get_batch(&self, batch_id: String) -> Result<Option<ClientBatch>, SawtoothClientError> {
         let url = format!("{}/batches/{}", &self.url, &batch_id);
 
-        if let Some(batch) = get::<Batch>(&url).unwrap() {
-            Ok(Some(batch.into()))
-        } else {
-            Ok(None)
-        }
+        Ok(get::<Batch>(&url)?.map(|batch| batch.into()))
```

Comment:
It looks like this update got added to the wrong commit; it should probably be in the previous one where the `get` function is introduced.

Reply:
Fixed